### PR TITLE
PAQ: Changed "Software AG Designer" to "Apama Plugin for Eclipse"

### PR DESCRIPTION
**Pull request by kailol-sag**
... and also renamed the shortcode file.

There are still other occurrences of "Software AG" such as the links to the Apama documentation. But this can only be changed when the Apama doc is available in Markdown and when I know the URL of the new doc website.

Similar changes (but we do not require a shortcode there) have been done on https://github.com/Cumulocity-IoT/c8y-docs-ops/pull/906 ,  which links to the #apama-plugin anchor defined in this PR.

### CODE REVIEW

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@kailol-sag @skom-c8y When is this change supposed to be merged?

File: themes/c8ydocs/layouts/shortcodes/apama-plugin.html [hunk 1]
```diff
@@ -0,0 +1 @@
+{{- "Apama Plugin for Eclipse" -}}
```
Comment (skom-c8y) [inline themes/c8ydocs/layouts/shortcodes/apama-plugin.html]: @kailol-sag  apama-plugin short code seems too generic and can be confused with other plugins like EPL plugin, Java plugin, connectivity plugin etc. can we instead use something like apama-plugin-for-eclipse ?

**Comment by skom-c8y** [comment]
> @kailol-sag @skom-c8y When is this change supposed to be merged?

we are actively working on other doc topics, and we should be able to merge these in over the next few weeks.

File: themes/c8ydocs/layouts/shortcodes/apama-plugin.html (hunk 1 above)
Comment (kailol-sag) [reply themes/c8ydocs/layouts/shortcodes/apama-plugin.html]: @skom-c8y Ah, you're right about the confusion. I wanted to use something short. But I'll change this to apama-plugin-for-eclipse as suggested.

**Comment by BeateRixen** [comment]
@skom-c8y This PR is only about the name change from Software AG Designer to Apama Plugin for Eclipse. I wonder if we can already merge this as we would like to reduce the number of SAG references throughout the documentation as much as possible until the end of the year. I know that there is more to do with all the links to the Apama documentation but we could implement these changes in a separate second step.

**Review by skom-c8y — APPROVED** [review]

**Comment by skom-c8y** [comment]
> @skom-c8y This PR is only about the name change from Software AG Designer to Apama Plugin for Eclipse. I wonder if we can already merge this as we would like to reduce the number of SAG references throughout the documentation as much as possible until the end of the year. I know that there is more to do with all the links to the Apama documentation but we could implement these changes in a separate second step.

done

**Pull request merged**